### ISSUE TITLE
Include homebrew release

### ISSUE THREAD

**Issue by aegershman** [pivotal/hammer#14]
In order to consume `hammer` updates from the mac packagemanager homebrew, it'd be nifty if `hammer` published a consumable homebrew repo as part of the [goreleaser process](https://github.com/pivotal-cf/om/blob/master/.goreleaser.yml#L31-L38).

[pivotal-cf/om does a similar thing](https://github.com/pivotal-cf/om/blob/master/om.rb)

thanks for considering it; feel free to close whenever 👍

**Comment by edwardecook** [comment]
Good suggestion, thanks! I hadn't realised it was relatively nicely built into goreleaser.

**Comment by edwardecook** [comment]
Added homebrew releasing and released v0.6.0 which can be installed with:
```
brew tap pivotal/hammer https://github.com/pivotal/hammer
brew install hammer
```

Again thanks for the suggestion, feel free to re-open this if there is anything else related.
